.travis.yml

```diff
@@ -1,5 +1,14 @@
 sudo: false
 language: d
+d:
+  - dmd-beta
+  - dmd
```

Comment:
Would need to change the makefile and how testing is done to support it. Possibly by adding a testing into the dub config.